data/src/main/java/com/globant/equattrocchio/data/response/Result.java

```diff
@@ -8,13 +8,13 @@
 public class Result {
```

Comment:
You should add names more descriptive, and the name format for response class is XXXResponse